.github/workflows/test-julia.yml

```diff
@@ -0,0 +1,36 @@
+name: CI
```

Comment:
We should either have the names reflect julia/python, or combine the jobs into one workflow called "CI". I'm leaning slightly toward the latter, any thoughts?

Reply:
Oops, I guess I forgot to push that change. I had changed it to `Test` locally, but never committed/pushed.

Combined testing jobs are fine with me. So we would just have **test.yml**, with name `____`, with two jobs names e.g. `test-python` and `test-julia`?

Reply:
Yeah, sounds good! I suppose naming it `Test` also makes more sense than `CI`.